documentation/suggested-project-structure.md

```diff
@@ -0,0 +1,97 @@
+### Suggested Project Structure
+
+ * `server.js` is the main entry point.
+ * Think of the `resources` folder as your `routes`.
```

Comment:
Coming from the routing+controllers ideology, I think this reduces the initial barrier to entry. JSON:API enforces all routes, the idea being you only have a say in what the resources look like, so calling the folder `resources` is the correct name and saying `think of it as a routes folder` is the best way to get people onboard quickly. A similar thing applies to `handlers` - they're something I've introduced that require a special name because all handlers must conform with a spec, you can't simply expose funky custom functionality and expect it to work.